### PR TITLE
Switch to cross-compilation for `linux_aarch64` and `linux_ppc64le`

### PR DESCRIPTION
**Pull request by bdice**
This PR switches to cross-compilation for `linux_aarch64` and `linux_ppc64le`.

See https://github.com/conda-forge/cuda-feedstock/issues/25

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin , please re-render

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,7 +49,8 @@ test:
     - test -f $PREFIX/compute-sanitizer/compute-sanitizer                           # [linux]
     - if not exist %LIBRARY_PREFIX%\compute-sanitizer\compute-sanitizer.exe exit 1  # [win]
     - if not exist %PREFIX%\Scripts\compute-sanitizer.bat exit 1                    # [win]
-    - compute-sanitizer --version
+    - {{ exists }} compute-sanitizer
+    - compute-sanitizer --version                                                   # [build_platform == target_platform]
```
Comment (jakirkham) [inline recipe/meta.yaml:53]: As we cannot run `compute-sanitizer` when cross-compiling, have modified the test above to simply check the existence of the command on all platforms. So there is some test that the executable exists even when cross-compiling

Then limited running `compute-sanitizer` to native builds

**Review by bdice — COMMENTED** [review]
@jakirkham I can’t approve my own PR but your changes look fine to me.

**Comment by jakirkham** [comment]
Alex mentioned offline he is having Internet trouble today

Will go ahead and merge to get this in

That said, happy to follow up on anything after as needed

**Pull request merged**